### ISSUE TITLE
Veyon + UWF: gone after 15 minutes

### ISSUE THREAD

**Issue by JohnAxae** [veyon/veyon#518]
I installed Veyon 4.1.8 and UWF as an alternative for Faronics Deepfreeze in a school, but I have a problem, the computer is not visible after 15 minutes.

I upgraded Veyon to 4.2.0, with that version it doesn't start at all if UWF is enabled,
when UWF is disabled the service starts and the computer is visible.
(I have not looked at how long the computer is visible with 4.2.0).

Server: Windows 2012 RC2
Clients: Windows 10 1809

**Comment by tobydox** [comment]
May be a problem with log files? Can you try to disable logging at all, i.e. change the log level to "Nothing" on all affected computers? Are the veyon-service and veyon-server processes still running once a computer is not visible any longer?

**Comment by JohnAxae** [comment]
I saw that the computer becomes "offline" when the computer is not used for a certain amount of time, and if a user interacts with the computer, it comes back "online".

**Update on UWF-problem**
I checked it today, if the service uses the default settings, it will crash and will not try to start again, if uwf is on. So if I changed it to a delayed start, it starts and they mostly come online.

**Comment by tobydox** [comment]
That's interesting! Is it related to screen blanking after 15 minutes or some other kind of power saving mechanism? Do sessions get locked automatically after inactivity?

**Comment by JohnAxae** [comment]
All the power saving settings are disabled, so no automatic sleep or blanking out screen/screensavers.

**Comment by JohnAxae** [comment]
Now they only show online when UWF is deactivated.
I enabled on some computers the option "Use alternative user authentication mechanism", but that doesn't solve it either.

I added the Veyon registry keys and folder to the exclusions and added the ports to our third-party firewall. But even that doesn't solve anything.

Some computers on the older version of Veyon still show up, and even a test computer I have set up close to me, works perfectly when UWF is on, on the latest build.
And even weirder is that when I close the Master and reopen it, the only desktop that is visible is the test computer..
